### ISSUE TITLE
Does not play sound

### ISSUE THREAD

**Issue by junglerobba** [flathub/com.slack.Slack#127]
Since the latest update I've noticed that my notifications were silent and the speaker test in audio settings also does not play any sound.

<details>
<summary>Full log looks like this:</summary>

```
~ > com.slack.Slack  
[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/extra/slack/usr/lib/slack/slack --type=zygote
Initializing local storage instance
(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.
(node:2) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `slack --trace-deprecation ...` to show where the warning was created)
/usr/lib/x86_64-linux-gnu/gio/modules/libgiolibproxy.so: undefined symbol: g_task_set_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiolibproxy.so
/usr/lib/x86_64-linux-gnu/gio/modules/libgiognomeproxy.so: undefined symbol: g_task_set_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiognomeproxy.so
/app/extra/slack/usr/lib/slack/../x86_64-linux-gnu/libgnutls.so.30: version `GNUTLS_3_6_3' not found (required by /usr/lib/x86_64-linux-gnu/gio/modules/libgiognutls.so)
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiognutls.so
Gtk-Message: 15:19:14.079: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:19:14.079: Failed to load module "canberra-gtk-module"
GLib-GIO-Message: 15:19:14.089: Using the 'memory' GSettings backend.  Your settings will not be saved or shared with other applications.
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)

(slack:2): Gtk-WARNING **: 15:19:15.886: GModule (/usr/lib/x86_64-linux-gnu/gtk-3.0/3.0.0/immodules/im-fcitx.so) initialization check failed: GLib version too old (micro mismatch)

(slack:2): Gtk-WARNING **: 15:19:15.886: Loading IM context type 'fcitx' failed
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
```

</details>
But I think this is probably the relevant line, especially as it appears right when opening the audio settings: 

```
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/x86_64-linux-gnu/alsa-lib/libasound_module_pcm_pulse.so
```

```
~ > flatpak info com.slack.Slack                    

Slack - Chat with your team

          ID: com.slack.Slack
         Ref: app/com.slack.Slack/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 4.15.0
     License: LicenseRef-proprietary
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 7.9 MB
     Runtime: org.freedesktop.Platform/x86_64/20.08
         Sdk: org.freedesktop.Sdk/x86_64/20.08

      Commit: 0fd72f49036de5fbe5e57a326e9336db16353330fbea87abf5c7dc5d83098da7
      Parent: 8bd61d0ff857bbd51d494c0b79c07dbdc211bec25a06b6beee1f539511a43e64
     Subject: Switch apply_extra to snap (1d9ec032)
        Date: 2021-06-28 09:34:17 +0000
```

**Comment by Mihai-B** [comment]
I am having the same problem. I cannot hear somebody in a conversation and they can't hear me either. Only happend since the last update

**Comment by schemar** [comment]
Same

**Comment by mdontu-bd** [comment]
Aside from sound not working, my mouse cursor appears to be using the default X11 theme.

**Comment by schemar** [comment]
My taskbar also doesn’t show the icon anymore and the context menu of the taskbar icon has blocks instead of readable characters.

**Comment by nhubaotruong** [comment]
Also ibus input (I use ibus-bamboo for Vietnamese input) not working as well, not sure about other ibus methods

**Comment by Mihai-B** [comment]
is there a way we can revert back to older version from flathub until this is fixed?

**Comment by junglerobba** [comment]
Fixed by e66db912314cf3cdadf796743502a3154a90d4bf
